src/Microsoft.Identity.Web/WebAppAuthenticationBuilderExtensions.cs

```diff
@@ -100,101 +95,175 @@ public static partial class WebAppAuthenticationBuilderExtensions
 
             builder.AddOpenIdConnect(openIdConnectScheme, options => { });
             builder.Services.AddOptions<OpenIdConnectOptions>(openIdConnectScheme)
-                .Configure<IServiceProvider>((options, serviceProvider) =>
+                .Configure<IServiceProvider, IOptions<MicrosoftIdentityOptions>>((options, serviceProvider, microsoftIdentityOptions) =>
            {
```

Comment:
This brace needs to move one space to the right and then apply the VS formatting to fix the below spacing and indent changes.
```suggestion
            {
```